pom.xml

```diff
@@ -446,9 +446,9 @@ under the License.
 
     <modernizer-maven.version>2.3.0</modernizer-maven.version>
 
-    <pac4j.version>5.2.1</pac4j.version>
+    <pac4j.version>5.3.1</pac4j.version>
 
-    <cas.version>6.5.0-RC4</cas.version>
+    <cas.version>6.5.1-SNAPSHOT</cas.version>
```

Comment:
Note that this version bump is necessary because of:
https://github.com/spring-cloud/spring-cloud-config/issues/1645 

This will get better in 6.6.x